src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -26,48 +27,54 @@ import Button from "components/atoms/Button";
 import "./UserProfileModal.scss";
 
 type PropTypes = {
```

Comment:
Prefer `interface` for props:

```suggestion
export interface UserProfileModalProps {
```